tests/gad_test.cpp

```diff
@@ -528,35 +528,196 @@ namespace OxTS
 			}
 		}
 
+		namespace GadWheelSpeed
+		{
+			BOOST_AUTO_TEST_CASE(GadWheelSpeed_WheelSpeedFw)
+			{
+				OxTS::GadWheelSpeed g(0);
+
+				g.SetWheelSpeedFw(1.99);
+
+				BOOST_CHECK(g.GetWheelSpeedFw() == 1.99);
+			}
+
+			BOOST_AUTO_TEST_CASE(GadWheelSpeed_WheelSpeedFwVar)
+			{
+				OxTS::GadWheelSpeed g(0);
+
+				g.SetWheelSpeedFwVar(1.99);
+
+				BOOST_CHECK(g.GetWheelSpeedFwVar() == 1.99);
+			}
+
+			BOOST_AUTO_TEST_CASE(GadWheelSpeed_WheelWheelSpeedCount)
```

Comment:
WheelWheelSpeed?

Reply:
Updated, typo from lots of 'wheel' changes :D